<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherData</class>
 <widget class="QMainWindow" name="WeatherData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Weather Station</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="updateValues">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Values</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>127</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="temperatureLabel">
         <property name="text">
          <string>Temperature</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="humidityLabel">
         <property name="text">
          <string>Humidity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pressureLabel">
         <property name="text">
          <string>Pressure</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QDoubleSpinBox" name="temperatureDoubleSpinBox"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="humidityDoubleSpinBox"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="pressureDoubleSpinBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
